docs/os/windows/windows-tools41.md

```diff
@@ -0,0 +1,292 @@
+Qubes Windows Tools
+===================
+
+Qubes Windows Tools (QWT) are a set of programs and drivers that provide integration of Windows 7, 10 and 11 AppVMs with the rest of the Qubes system. Currently the following features are available for Windows VMs after installation of those tools:
+
+-   **Qubes Video Driver** - provides for the seamless GUI mode that integrates windows apps onto the common Qubes trusted desktop (currently only for Windows 7)
+-   **Clipboard sender/receiver** - Support for [secure clipboard copy/paste](https://www.qubes-os.org/doc/copy-paste/) between the Windows VM and other AppVMs
+-   **File sender/receiver** - Support for [secure file exchange](https://www.qubes-os.org/doc/copying-files/) between the Windows VM and other AppVMs
+-   **Copy/Edit in Disposable VM** - Support for editing files in DisposableVMs as well as for `qvm-run` and generic `qrexec` for the Windows VM (e.g. ability to run custom service within/from the Windows VM)
+-   **Xen PV drivers** for Windows that increase performance compared to QEMU emulated devices
+
+Below is a breakdown of the feature availability depending on the windows version:
+
+|             Feature                  |  Windows 7 x64 | Windows 10/11 x64 |
+| ------------------------------------ | :------------: | :---------------: |
+| Qubes Video Driver                   |        +       |         -         |
+| Qubes Network Setup                  |        +       |         +         |
+| Private Volume Setup (move profiles) |        +       |         +         |
+| File sender/receiver                 |        +       |         +         |
+| Clipboard Copy/Paste                 |        +       |         +         |
+| Application shortcuts                |        +       |         +         |
+| Copy/Edit in Disposable VM           |        +       |         +         |
+| Block device                         |        +       |         +         |
+| USB device                           |        +       |         +         |
+| Audio                                |        +       |         +         |
```

Comment:
It would be useful to add few notes:
* Block device support requires Xen PV Disk Drivers installed
** USB and Audio support requires R4.1 and available even w/o QWT installation